api/source/job_control/add_media_to_job.rst

```diff
@@ -58,6 +58,18 @@ specify the Content-Length in the header. File size is limited to 10 gb.
 |                  | `Example`        | ``media_url=http%3A%2F%2Fwww.domain.com%2Fvideo.mp4``     |
 +------------------+------------------+-----------------------------------------------------------+
 
+**Query String Parameters** — Optional `(when adding media from URL)`
+
++------------------+------------------------------------------------------------------------------+
+| Name             | Details                                                                      |
++==================+==================+===========================================================+
+| is_duplicate     | `Description`    | Bypass the ducplicate detection settings                  |
+|                  +------------------+-----------------------------------------------------------+
+|                  | `Allowed Values` | Boolean                                                   |
```

Comment:
Is boolean case insensitive? Maybe you can make this explicit

Reply:
It is not case-sensitive, addressed